src/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -424,7 +424,43 @@ private function addHttpCacheSection(ArrayNodeDefinition $rootNode): void
                                 ->end()
                                 ->arrayNode('http_tags')
                                     ->canBeDisabled()
-                                    ->info('Enable support for Cache-Tags invalidation.')
+                                    ->info('Enable support for Cache-Tag invalidation.')
+                                ->end()
+                                ->arrayNode('purger')
+                                    ->addDefaultsIfNotSet()
+                                    ->info('Define the purger configuration.')
+                                    ->children()
+                                        ->scalarNode('dsn')
+                                            ->defaultValue('')
+                                            ->info('Purger dsn')
+                                        ->end()
+                                        ->scalarNode('path')
+                                            ->defaultValue('')
+                                            ->info('Purger API path')
+                                        ->end()
+                                        ->scalarNode('token')
+                                            ->defaultValue('')
+                                            ->info('JWT token to use if the provider API if secured')
+                                        ->end()
+                                    ->end()
+                                ->end()
+                                ->scalarNode('type')
+                                    ->defaultValue('')
+                                    ->info('Cache provider to use (e.g. souin, varnish, whatever)')
+                                ->end()
+                            ->end()
+                        ->end()
+                        ->arrayNode('validation')
+                            ->addDefaultsIfNotSet()
+                            ->info('Enable the tags-based cache validation system.')
+                            ->children()
+                                ->scalarNode('header')
+                                    ->defaultValue('Cache-Tag')
```

Comment:
should be `Cache-Tags` or it is a BC as we had this as default in 2.6. We can deprecate `Cache-Tags` in favor of `Cache-Tag` in 2.7 if you all agree but as there is no real specification I'm unsure of the correct answer.

Reply:
The cloudflare specification would be the correct answer. Anyway, I'll write a Cloudflare V4 provider and revert the Cache-Tag into Cache-Tags for the existing one.
IMHO keep the Cache-Tags header sucks because it's not maintained by Cloudflare and the "new" specification would be the only one we have to follow.

Reply:
yes I know the only issue is that it breaks current apis so we can't just change it we need to deprecate it first